### ISSUE TITLE
Publication Quality Graphics with R: Time Series Plots

### ISSUE THREAD

**Issue by smcclatchy** [smcclatchy/studyGroup#15]
We'll use the ggplot2 package in R to produce a publication-quality line plot from time series data. In this edition, you'll learn to: 
- Load data from a URL.
- Convert date objects and formats.
- Log-transform the scale of a plot.
- Specify breaks in the x and y axes.
- Use plot annotations, custom axis labels, and axis themes.
- Save plot as a PDF.

We'll meet on Wednesday, July 20 in B53 ERB 2535 from 9:30 to 10:30am.
https://myjax-p.jax.org/Facilities/Maps/ResearchLabMap.pdf

Bring a laptop with R and RStudio installed. Beginners are welcome!
https://cran.r-project.org/
https://www.rstudio.com/

**Comment by smcclatchy** [comment]
You can find the tutorial and code here:

**Tutorial**
https://github.com/smcclatchy/R-graphics/blob/master/line.Rmd

**Code**
https://github.com/smcclatchy/R-graphics/blob/master/time-series-line-plot.R

**Comment by smcclatchy** [comment]
Mistakes can lead to [accidental aRt](http://accidental-art.tumblr.com/).
